ios/NativeBridge.swift

```diff
@@ -0,0 +1,220 @@
+            )
+            resolve(credentialsManager.store(credentials: credentials))
+        } else {
+            reject(NativeBridge.credentialsManagerErrorCode, "Incomplete information provided for credentials - 'expiresIn' not found", NSError.init(domain: NativeBridge.credentialsManagerErrorCode, code: -99999, userInfo: nil));
+        }
+    }
+    
+    @objc public func getCredentials(scope: String?, minTTL: Int, parameters: [String: Any], forceRefresh: Bool, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
+        if(forceRefresh) {
+            credentialsManager.renew { result in
+                switch result {
+                case .success(let credentials):
+                    resolve(credentials.asDictionary())
+                case .failure(let error):
+                    reject(error.getReactNativeErrorCode(), error.errorDescription, error)
+                }
+            }
+        } else {
+            credentialsManager.credentials(withScope: scope, minTTL: minTTL, parameters: parameters) { result in
+                switch result {
+                case .success(let credentials):
+                    resolve(credentials.asDictionary())
+                case .failure(let error):
+                    reject(error.getReactNativeErrorCode(), error.errorDescription, error)
+                }
+            }
+        }
+    }
+    
+    @objc public func hasValidCredentials(minTTL: Int, resolve: RCTPromiseResolveBlock) {
+        resolve(credentialsManager.canRenew() || credentialsManager.hasValid(minTTL: minTTL))
+    }
+    
+    @objc public func clearCredentials(resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+        resolve(credentialsManager.clear())
+    }
+    
+    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?, evaluationPolicy: Int) {
+        let titleValue = title ?? "Please authenticate to continue"
+        let policyValue = self.convert(policyInt: evaluationPolicy)
+        self.credentialsManager.enableBiometrics(withTitle: titleValue, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle, evaluationPolicy: policyValue)
+    }
+
+    func convert(policyInt: Int) -> LAPolicy {
+        if (policyInt == 2) {
+            return LAPolicy.deviceOwnerAuthentication
+        }
+        return LAPolicy.deviceOwnerAuthenticationWithBiometrics
+    }
+}
+
+
+extension Credentials {
+    func asDictionary() -> [String: Any] {
+        return [
+            NativeBridge.accessTokenKey: self.accessToken,
+            NativeBridge.tokenTypeKey: self.tokenType,
+            NativeBridge.idTokenKey: self.idToken,
+            NativeBridge.refreshTokenKey: self.refreshToken as Any,
+            NativeBridge.expiresInKey: floor(self.expiresIn.timeIntervalSince1970),
+            NativeBridge.scopeKey: self.scope as Any
+        ]
+    }
+}
+
+extension WebAuthError {
+    func getReactNativeErrorCode() -> String {
```

Comment:
The [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/) state that methods that do not have side effects should read as nouns:

<img width="747" alt="Screenshot 2023-06-23 at 10 51 17 AM" src="https://github.com/auth0/react-native-auth0/assets/5055789/839732d8-e086-416c-be55-29bb0e0da89c">

`getReactNativeErrorCode()` being a pure function, should instead be named something like `reactNativeErrorCode()`. Or simply make it a computed property, since it takes no parameters. In which case it should still read as a noun:

<img width="719" alt="Screenshot 2023-06-23 at 10 51 31 AM" src="https://github.com/auth0/react-native-auth0/assets/5055789/a7c04cd8-a1b9-4496-aa1a-090fa48acd6b">

Reply:
The Java dev in me got the best of me there 😅